charts/fluent-bit/values.yaml

```diff
@@ -0,0 +1,98 @@
+---
+version: v1  # service version, good practice to label every k8s resource with a version if we want to use istio traffic routing capabilities in the future
+
+nameOverride: "fluent-bit"  # replace this by your service name
+fullnameOverride: ""  # fill if you whish to override the generatedChartFullName
+name: &nameOverride
+
+# Controls if prometheus scrapping should be enabled for this service
+prometheus:
+  enabled: true  # enables/disable scraping of prometheus metrics
+  path: "/api/v1/metrics/prometheus"  # path for scrapping prometheus metrics
+  port: 2020  # port were the metrics web server is running
+  scheme: "tcp"  # tcp connection scheme
```

Comment:
tcp scheme is not supported only http or https

Reply:
I will update to http